src/Options.js

```diff
@@ -58,22 +68,69 @@ Options.prototype.extend = function(extension) {
     return new Options(data);
 };
 
+function sizeAtStyle(size, style) {
+    return style.size < 2 ? size : sizeStyleMap[size - 1][style.size - 1];
```

Comment:
No because style.size might be 0 (DISPLAY).

It would be equivalent to `sizeStyleMap[size - 1][Math.max(0, style.size - 1)]` but I find this easier to understand